react/components/EXPERIMENTAL_Table/types.ts

```diff
@@ -23,3 +32,43 @@ export type Column = {
 export type E2ETestable<T = string> = {
   testId?: T
 }
+
+export interface HasMotion {
+  motion?: ReturnType<typeof useTableMotion>
+}
+
+/** Shorthand for RefForwardingComponent */
+export type RFC<T, P = {}> = React.RefForwardingComponent<T, P>
+
+/** Defines a exported foward ref with composites */
+export type ComposableWithRef<
+  Ref = HTMLElement,
+  Props = PropsWithChildren<{}>,
+  Composites = {}
+> = ForwardRefExoticComponent<Props & RefAttributes<Ref>> & Partial<Composites>
+
+/** Type with children that receives render-props */
+export type RenderProps<P = {}, T = Record<string, unknown>> = P & {
+  children?: (data: T) => ReactNode
+}
+
+/** Ref foward component with render props 
+export interface RFCRP<T, P = {}, R = Record<string, unknown>> {
+  (props: RenderProps<P, R>, ref: React.Ref<T>): React.ReactElement | null
+}
+*/
```

Comment:
If its not being used it should be removed

Reply:
sure. nice catch